src/nbl/asset/utils/CMeshManipulator.cpp

```diff
@@ -441,10 +482,7 @@ static bool cmpVertices(ICPUMeshBuffer* _inbuf, const void* _va, const void* _vb
 //! Creates a copy of a mesh, which will have identical vertices welded together
 core::smart_refctd_ptr<ICPUMeshBuffer> IMeshManipulator::createMeshBufferWelded(ICPUMeshBuffer *inbuffer, const SErrorMetric* _errMetrics, const bool& optimIndexType, const bool& makeNewMesh)
```

Comment:
this whole function is extremely slow btw, i dont know if its even worth it  XD its O(vtxCount^2)

Reply:
and memcpy in every iteration

Reply:
its a future todo for @Przemog1 to use the spatial hash he already uses for smooth normals